### PR TITLE
Update `.gitattributes`

### PR DESCRIPTION
**Pull request by tfirdaus**

### CODE REVIEW

**Comment by szepeviktor** [comment]
Here is how to test: `git archive HEAD | tar --list --exclude="*/*"`

**Review by szepeviktor — APPROVED** [review]

File: .gitattributes [hunk 1]
```diff
@@ -9,13 +9,13 @@
...
 /docker-compose.yml export-ignore
 /docs export-ignore
 /examples export-ignore
 /phpcs.xml.dist export-ignore
-/phpstan.neon export-ignore
+/phpstan.neon.dist export-ignore
 /phpunit.xml.dist export-ignore
-/README.md export-ignore
 /tests export-ignore
 /tmp export-ignore
```
Comment (szepeviktor) [inline .gitattributes:21]: tmp could also go.

Comment (tfirdaus) [reply .gitattributes:21]: You're right :) thanks

File: .gitattributes [hunk 2]
```diff
@@ -9,13 +9,12 @@
 /.github export-ignore
 /.gitignore export-ignore
 /.vscode export-ignore
-/bin export-ignore
+/CODE_OF_CONDUCT.md export-ignore
+/codecov.yml export-ignore
 /docker-compose.yml export-ignore
 /docs export-ignore
```
Comment (szepeviktor) [inline .gitattributes:15]: We don't have `docs/` yet.

File: .gitattributes [hunk 3]
```diff
@@ -9,13 +9,12 @@
...
 /examples export-ignore
```
Comment (szepeviktor) [inline .gitattributes:16]: Nor `examples/`.

**Comment by szepeviktor** [comment]
Currently exported files.
```
LICENSE
README.md
app/
composer.json
inc/
```

File: .gitattributes (hunk 2 above)
Comment (tfirdaus) [reply .gitattributes:15]: @szepeviktor but we will. So I think it's fine :)

**Pull request merged**